### PR TITLE
Contrast enhancements

### PR DESCRIPTION
**Pull request by faustinoaq**
Modify main-orange color to pass WCAG AA spec with large white text (the main color scheme of Amber)

### CODE REVIEW

**Comment by faustinoaq** [comment]
@eliasjpr @RyanScottLewis We are trying to pass WCAG AA spec here 👉 http://wave.webaim.org/report#/https://amberframework-pr-41.herokuapp.com/

**Comment by faustinoaq** [comment]
@RyanScottLewis Please, I need you run `npm install; npm run release` to generate new `main.bundle.css` 😅 to see changes on https://amberframework-pr-41.herokuapp.com/

**Comment by faustinoaq** [comment]
@eliasjpr I think this PR is ready to merge, please take a look 😄

**Comment by RyanScottLewis** [comment]
Heh sorry guys, I'm not used to this workflow. That's all the changes =]

File: src/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -20,7 +21,8 @@ small {
 }
 
 .topbar-nav .nav-link {
-    opacity: 0.9;
+    opacity: .9;
+    font-size: 1rem;
```
Comment (eliasjpr) [inline src/assets/stylesheets/main.scss]: This is too big. Can we keep the font size as it is currently?

Comment (RyanScottLewis) [reply src/assets/stylesheets/main.scss]: That's the smallest size that passes AA spec, however I've reverted it because it does look better as it was imo

**Pull request merged**